release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1379,6 +1379,30 @@ With this release, if you disable the `Build` cluster capability, the `builder`
 
 For more information, see xref:../installing/cluster-capabilities.adoc#build-config-capability_cluster-capabilities[Build capability].
 
+[discrete]
+[id="ocp-4-16-dal12-region_{context}"]
+=== dal12 region was added
+
+With this release, the `dal12` region has been added to the {ibm-power-name} VS Installer.
+
+[discrete]
+[id="ocp-4-16-power-vs-regions_{context}"]
+=== Regions added to power virtual server
```

Comment:
```suggestion
=== Regions added to Power Virtual Server
```

Reply:
I couldn't find anywhere in the repo where this wasn't capitalized.

Reply:
If this is an IBM specific thing, it seems like IBM precedes Power Virtual Server `IBM Power Virtual Server`.